db/seeds.rb

```diff
@@ -24,6 +24,13 @@
     website_url: nil,
     menu_url: nil,
     square_location_id: 'E4R1NCMHG7B2Y'
+  },
+  {
+      seller_id: 'send-chinatown-love',
+      accept_donations: false,
+      sell_gift_cards: false,
+      founded_year: 2020,
+      square_location_id: "" # TODO
```

Comment:
This should probably be an ENV variable

Reply:
Should I also make the one for shunfa an ENV var?